.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
```

Comment:
The `env` is set to `es2024`, but the `ecmaVersion` in `parserOptions` is set to 12. These should be aligned to ensure compatibility. Consider setting `ecmaVersion` to 2024 if you intend to use ES2024 features.